extensions/tmtv/src/Panels/PanelPetSUV.tsx

```diff
@@ -135,6 +118,11 @@ export default function PanelPetSUV({ servicesManager, commandsManager }) {
 
     // update the displaySets
     displaySetService.setDisplaySetMetadataInvalidated(ptDisplaySet.displaySetInstanceUID);
+
+    // seems like for some reason crosshairs should get reset
```

Comment:
Can you comment on this - for example
// Crosshair position depends on the metadata values such as the positioning interaction
// between series, so when the metadata is updated, the crosshairs need to be reset.

Reply:
Thanks